MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/model/usermodel/Authorizations.java

```diff
@@ -0,0 +1,105 @@
+package com.mapbox.mapboxandroiddemo.model.usermodel;
+
+/**
```

Comment:
Remove file header.
[Turning off “created by” stamp when generating files in AS](http://stackoverflow.com/questions/20109203/android-studio-how-to-remove-update-the-created-by-comment-added-to-all-new-c)